lib/transform.js

```diff
@@ -3,7 +3,8 @@
 var _             = require('underscore')
   , anchor        = require('anchor-markdown-header')
   , updateSection = require('update-section')
-  , getHtmlHeaders = require('./get-html-headers');
+  , getHtmlHeaders = require('./get-html-headers')
+  , md            = require('markdown-to-ast');
```

Comment:
Just curious (I haven't looked into this too much), what were some of the factors that led to this markdown parser? Seems like there are a number of other options: `marked-ast`, `markdown-ast`, `markdown-to-ast` to name a few. Was this one easier to use, faster, more well-documented...)